### PR TITLE
Ability to pass additional environments to calico-node and typha

### LINKED ISSUE #2: expose felix configuration settings in values.yaml?
Hi,

I installed calico using this cart first time on my test lab on AWS. On AWS, the src/dest check needs to be disabled for instances. But settings can't be set on autoscale groups.

For this scenario, calico can disable the source/destination check on the instance.

I guess I could set this by define `FELIX_AWSSRCDSTCHECK=Disable` on the calico-node, but there is no chance to define this inside the chart.

See: https://docs.projectcalico.org/reference/resources/felixconfig#aws-iam-rolepolicy-for-source-destination-check-configuration

### PR DESCRIPTION
**Pull request by jkroepke**
Fixes: #2 

Additionally, I fixed the duplicate port definition (IDE marks the as error).

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -7,8 +7,18 @@
...
 
+podCIDR: ""
 #podCIDR: "192.168.0.0/16"
 typha: false
 flexVolumePluginDir: "/usr/libexec/kubernetes/kubelet-plugins/volume/exec"
 vethMtu: 1440
 autoHostEndpoints: false
+
+typha_env:
+  # TYPHA_PROMETHEUSMETRICSPORT: "9093"
```
Comment (invidian) [inline values.yaml]: Can we make it consistent with node settings and have `typha.env` please? I like having `node` and `typha` top level keys, as they can be then extended in the future.

**Review by invidian — CHANGES_REQUESTED** [review]
Thanks for the PR! I have just one suggestion, otherwise it looks good to me :+1:

File: values.yaml (hunk 1 above)
Comment (jkroepke) [reply values.yaml]: Yes, but this is breaking, since `typha` is already a boolean. See line above

Comment (invidian) [reply values.yaml]: Right. Hmm, I think it should be fine to change it. I don't expect there is many users of this chart right now and the sooner that happens, the better.

```yaml
typha:
  enabled: false
  env:
```

With that, let's bump the chart version to `0.3.0`.

Comment (jkroepke) [reply values.yaml]: Good choice! I love it.

**Review by invidian — APPROVED** [review]
LGTM, Thanks again @jkroepke

**Comment by invidian** [comment]
```
NAME            CHART VERSION   APP VERSION     DESCRIPTION
flexkube/calico 0.3.0           3.16.5          A Helm chart for Calico
```

New chart version is now available.

**Pull request merged**